requirements.txt

```diff
@@ -9,4 +9,7 @@ sphinx<3.5.0
 sphinx_rtd_theme
 lxml==4.6.3
 rst2pdf
-ninja
\ No newline at end of file
+ninja
+
+# Needed for test/Parallel/failed-build/failed-build.py
+psutil
```

Comment:
I don't really like that these are in `requirements.txt`, that's supposed to be runtime requirements. The only requiremen to *run* SCons is ninja (and then only if you explicitly invoke it).  Things needed to build the docs, run the tests, etc. are development requirements, and should go into a separate `requirements_dev.txt` (or, as someone has suggested, into `setup.py` set up so you can run `python setup.py develop` to get a dev env).

Reply:
Runtime requirements are in the setup.py logic.  requirements.txt is for development work..  Either that or we need to split it out...